examples/android/client/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Flower</string>
+    <string name="notification_channel_id">1002</string>
```

Comment:
Why 1002? Whats the significance of this?

Reply:
This variable acts as an ID for the notification delivered by the Background Worker routine. The ID acts as a unique identifier for the notification channel. Any constant number would work fine. For simplicity I will change this number to 1 or 2.

Reply:
@Shaz-hash got it! No need to change it